test.js

```diff
@@ -0,0 +1,27 @@
+
+var test = require('tape')
+var mpg123 = require('./lib/audio_mpg123.node')
+
+test('exports functions', function (t) {
+  t.plan(1)
+  console.log(mpg123)
+  t.true(
+    mpg123
+    && mpg123.create
+    && mpg123.open
+    && mpg123.close
+    && mpg123.write
+    && mpg123.flush
+  )
+})
+
+test('opens mpg123', function (t) {
```

Comment:
The reason the old test never checked if the binding could open, was because the test was designed for Travis during `audio-speaker` tests. Travis doesn't have the appropriate environment to open an instance of mpg123. If you could test this, try find a way around it or make another test for Travis (add it to the commands Travis will run). That would be great!